dds_registration/models.py

```diff
@@ -193,6 +195,33 @@ class Payment(Model):
     # invoice or a receipt
     data = models.JSONField(help_text="Read-only JSON object", default=dict)
 
+    def get_data(self):
```

Comment:
No JSON anything is needed, all of that is done automatically. You are making this much more complicated than it needs to be. Please see https://github.com/Depart-de-Sentier/dds_registration/pull/133/ for a clean way to fix this issue.